devops/jobs/ReplaceUsernames.groovy

```diff
@@ -0,0 +1,89 @@
+package devops.jobs
+
+/* JenkinsPublicConstants can be found in src/main/groovy/org/edx/jenkins/dsl*/
+/* It contains values and methods that are used repeatedly through the dsl jobs */
+import org.yaml.snakeyaml.Yaml
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_WORKER
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.GENERAL_SLACK_STATUS
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_GITHUB_BASEURL
+
+/*
+Need to manually add files with the following details to $WORKSPACE/username-replacement-secure/$ENVIRONMENT.yml:
```

Comment:
Is it common to have to manually update a yml file for a jenkins job? Seems odd to me but maybe this is a pattern i just haven't seen

Reply:
We don't currently have a good automation story for getting secrets available on jenkins while keeping them secure.  We use the jenkins secureBindings for this but don't have a way of adding new secure content automatically.

Reply:
@tuchfarber can you update this comment to indicate that these are the contents of the credentialBinding.